test/test_transforms_tensor.py

```diff
@@ -188,6 +188,10 @@ def test_crop(self):
             'crop', 'RandomCrop', fn_kwargs=fn_kwargs, meth_kwargs=meth_kwargs
         )
 
+        # Test transforms.functional.crop including outside the image area
+        fn_kwargs = {"top": -2, "left": 8, "height": 4, "width": 5}
```

Comment:
Can you add more tests exercising other use-cases, like when `top` is negative and `height + top` is larger than the image size?

Reply:
Added tests considering all directions!